examples/clientApplication/client/index.js

```diff
@@ -3,8 +3,8 @@ $(document).ready(function() {
         var setCount = 0;
         setCount += prePopulateValues(e, 'configure-login-service-dialog-clientId', 'clientApplication');
         setCount += prePopulateValues(e, 'configure-login-service-dialog-secret', '12345');
-        setCount += prePopulateValues(e, 'configure-login-service-dialog-baseUrl', 'http://localhost:3100');
```

Comment:
I must have overlooked it. I was using port 3100 for the clientApplication and port 3200 for the resourceServer.